plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TransactionLogTail.java

```diff
@@ -95,4 +139,68 @@ public long getRetainedSizeInBytes()
                 + SIZE_OF_LONG
                 + estimatedSizeOf(entries, Transaction::getRetainedSizeInBytes);
     }
+
+    // Load a section of transaction log JSON entries starting from `startVersion`(inclusive) up to the latest version
+    private static TransactionLogTail loadNewTail(
```

Comment:
I'm finding the input paramaters to this method hard to understand.
The code comment doesn't describe what should be populated in `version`.
One of the callers is using `startVersion` for `version` parameter and `startVersion + 1` for `startVersion` parameter.
Another is using 0 for both, so its confusing to figure out what the caller is supposed to populate.